example/src/main/res/layout/create_post_activity.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:background="@drawable/edit26"
```

Comment:
- please use better naming for icons (ic_edit, ic_close, ...)
- remove unused icons
- do those icons works on pre-lollipop devices?